app/javascript/dashboard/components/ChatList.vue

```diff
@@ -150,24 +150,16 @@ export default {
     },
     conversationList() {
       let conversationList = [];
+      const filters = this.conversationFilters;
       if (this.activeAssigneeTab === 'me') {
-        conversationList = this.mineChatsList.slice();
+        conversationList = [...this.mineChatsList(filters)];
```

Comment:
Please don't do this. This would change the array reference every time a change is happening.